app/modules/authentication/saga/api/confirmation.js

```diff
@@ -14,6 +14,7 @@ export const apiPostConfirmationSaga = function* (
     yield call(ConfirmationApi.post, email);
   }
   catch (error) {
-    // TODO
+    // TODO: fix saga error handling
+    throw error;
```

Comment:
What is the plan here? Dispatch an error action?

Reply:
This is not included in this branch yet, but that's the idea. If a request was successful, dispatch an API request success action, if not an API request failure action.